app/models/manageiq/providers/vmware/infra_manager/provision/cloning.rb

```diff
@@ -60,6 +56,17 @@ def prepare_for_clone_task
     clone_options
   end
 
+  def dest_folder
+    folder_id = get_option(:placement_folder_name)
+    return EmsFolder.find_by(:id => folder_id) if folder_id
+
+    host_dc = dest_host.parent_datacenter || dest_host.ems_cluster.parent_datacenter
+    proposed_folder = "Datacenters/#{host_dc.name}/vm/Discovered virtual machine"
+    EmsFolder.where(:name => 'Discovered virtual machine').detect do |f|
```

Comment:
@bzwei VMware provisioning requires that a folder be passed so my concern is that the "Discovered virtual machine" folder may not exist for some reason.  In that case we should drop back to use the root `vm` folder.

I think the lookup logic should be move into a new method that we can call with each path and it would return the folder or nil.

Something like:
vm_folder = "Datacenters/#{host_dc.name}/vm"
`find_dest_folder("#{vm_folder}/Discovered virtual machine") || find_dest_folder(vm_folder)`